functional/kata-monitor/run.sh

```diff
@@ -0,0 +1,245 @@
+#!/bin/sh
+#
+# Copyright (c) 2022 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test file will test kata-monitor for basic functionality (retrieve kata sandboxes)
+# It will assume an environment where:
+# - a CRI container manager (container engine) will be up and running
+# - crictl is installed and configured
+# - the kata-monitor binary is available on the host
+#
+
+CRI_RUNTIME=${CRI_RUNTIME:-"crio"}
+RUNTIME=${RUNTIME:-"containerd-shim-kata-v2"}
+KATA_MONITOR_BIN=${KATA_MONITOR_BIN:-$(type -p kata-monitor)}
```

Comment:
Unlikely though it is, this would fail if `/bin/sh` was zsh so I'd consider using `command -v kata-monitor` as the output of that command is consistent between shells.